mobile/remote

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -euo pipefail
+
+script_root="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+certificate="$script_root/tmp/certs/engflow.crt"
+key="$script_root/tmp/certs/engflow.key"
+
+if [[ -f "$certificate" && -f "$key" ]]; then
+  echo "Using EngFlow RBE"
+else
+  echo "EngFlow RBE certificate and key not installed in $certificate and $key"
+  echo "Contact Andrés Felipe Barco Santa <andres@engflow.com> for access to these files"
+  exit 1
+fi
+
+if [[ $OSTYPE == darwin* ]]; then
+  # TODO(jpsim): RBE fails with arm64 versions of Bazel on macOS
+  BAZELW_ARCH=amd64 ./bazelw "$@" \
```

Comment:
If, as per the comment on line 18, RBE fails with amd64, I'm surprised that we're setting BAZELW_ARCH=amd64 here. Can you say more abut this?

Reply:
I'll add more context.

What happens is that if you run a `bazel build` command on an M1 Mac host, at some point it tries to execute an arm64 binary of `protoc` on the amd64 remote exec machine, leading to [this error](https://envoy.cluster.engflow.com/invocation/30cc6fcc-8c30-46b0-b616-48af66cec8a9#console):

```
sandbox-exec: execvp() of 'bazel-out/.../com_google_protobuf/protoc' failed: Bad CPU type in executable
```

By forcing the host to use an amd64 binary of bazel (which can execute via Rosetta on an arm64 Mac), an amd64 `protoc` binary is used on the amd64 remote exec machine, preventing the error.

I might be misinterpreting what's happening here, but in the end forcing amd64 works.

Reply:
I'm working with @jayconrod from EngFlow over email to dig more into this.

Reply:
Oh, man! I read this like 5 times and FINALLY realized that "amd64" != "arm64". *facepalm*

Reply:
I filed https://github.com/envoyproxy/envoy/issues/24605 to track.